backend/src/test/java/ai/verta/modeldb/TestsInit.java

```diff
@@ -73,8 +73,11 @@ public class TestsInit {
   protected static HydratedServiceGrpc.HydratedServiceBlockingStub
       hydratedServiceBlockingStubClient2;
   protected static ArtifactStoreGrpc.ArtifactStoreBlockingStub artifactStoreBlockingStub;
+  private static Boolean setServerAndServiceRan = false;
 
   protected static void init() throws Exception {
+    if (setServerAndServiceRan) return;
```

Comment:
style nit: I would greatly prefer we have brackets around all blocks

Reply:
Changed it.